docs/reference/security/fips-140-compliance.asciidoc

```diff
@@ -8,59 +8,75 @@ government computer security standard used to approve cryptographic modules.
 {es} offers a FIPS 140-2 compliant mode and as such can run in a FIPS 140-2
 configured JVM.
 
-IMPORTANT: The JVM bundled with {es} is not configured for FIPS 140-2. You must 
-configure an external JDK with a FIPS 140-2 certified Java Security Provider.
+IMPORTANT: The JVM bundled with {es} is not configured for FIPS 140-2. You must
+configure an external JDK and it highly recommended to use a FIPS 140-2 certified Java Security Provider.
 Refer to the {es}
 https://www.elastic.co/support/matrix#matrix_jvm[JVM support matrix] for
-supported JVM configurations.
+supported JVM configurations. See https://www.elastic.co/subscriptions[subscriptions] for required licensing.
 
-After configuring your JVM for FIPS 140-2, you can run {es} in FIPS 140-2 mode by
-setting the `xpack.security.fips_mode.enabled` to `true` in `elasticsearch.yml`.
+Compliance with FIPS 140-2 is achieved by using only FIPS approved / NIST recommended cryptographic algorithms. This can be achieved by the following:
```

Comment:
This may be confusing and read as this is the only required step to achieve compliance. I think that it's important to mention that you also need the compliant JDK.